apps/bfd-model/bfd-model-codebook-data/pom.xml

```diff
@@ -15,6 +15,10 @@
 		Converts the CCW Data Dictionary codebook PDFs to XML.
 	</description>
 
+	<properties>
+		<generatedSources>${project.build.directory}/generated-sources/java</generatedSources>
+	</properties>
```

Comment:
I don't understand why this is needed only now. Is it actually needed?

Reply:
Looks like it's not